_config.yml

```diff
@@ -41,6 +41,10 @@ authors:
 
 disqus: devitconf
 
+og:
```

Comment:
`og` is a bit misleading. While is valid for the Open graph tags(FB) it's not appropriate for Twitter and other meta tags. We should change it to something more valid.